### DISCUSSION TITLE
Github Pages build fails with remote theme on SCSS undefined variable

### DISCUSSION THREAD

**Discussion by fscelliott** [mmistakes/minimal-mistakes#3104] [Q&A]
Env
---

Minimal Mistakes version: N/A?
Ruby gem or remote theme version: 
	I think I must be using the latest? In _config.yml I have:
             remote_theme: "mmistakes/minimal-mistakes" 
Jekyll version:
           My Gemfile.lock has jekyll (= 3.7.3) 
Git repository URL: https://github.com/fscelliott/docsdonewell 
GitHub Pages hosted (if yes provide URL to site): https://www.franceselliott.com/
Operating system: N/A

Reproduce
----

Sorry, I’m a hacker at best when it comes to coding and Jekyll.

Back in April I made this commit for a new blog post (no other changes in the commit) https://github.com/fscelliott/docsdonewell/tree/6b33989f808218fe7d7d5f431fd5fe5c6021c092 

And at that time GitHub Pages build started failing with this error:

Your SCSS file assets/css/main.scss has an error on line 20: Undefined variable: "$notice-background-mix".

I’m having trouble solving this (spent over 4 hrs).  Not sure what else to include but will be happy to include more info!   One of the things I did to troubleshoot was add my own [assets/css/main.scss](https://github.com/fscelliott/docsdonewell/commit/17f00219ff135a5683093aad5fbdbf14b9e4d285) but that didn't help.

It also fails in Travis CI with the same error with this .travis.yml:

```
language: ruby

cache: bundler 

before_install: gem update bundler

install:
  - bundle install

script:
  - bundle exec jekyll build
branches:
  only:
  - master
```

**Comment by mmistakes** [comment]
Looks like you customized the theme’s Sass files and removed some partials and variables in your local files. When doing that Jekyll will use your local Sass files instead of the theme’s and since it can’t find some variables it errors out. 

Compare your _variables.scss against what is bundled in the theme and you’ll see what I mean.

https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_variables.scss

**Reply by fscelliott** [reply]
thank you so much! Based on your comment, I reverted a PR that I'd asked someone on Fiverr to make to modify my fonts, and indeed, that solved my bug. Very very grateful for the advice! Now I'll just need to figure out how to get  Montserrat as my default font in a safer way than that Fiverr [PR](https://github.com/fscelliott/docsdonewell/pull/11/files). I'll work on it!
